spring-cloud-gcp-logging/src/main/java/org/springframework/cloud/gcp/logging/CompositeTraceIdExtractor.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.logging;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import com.google.common.collect.ImmutableList;
+
+import org.springframework.util.Assert;
+
+/**
+ * Delegates checking HTTP request to a list of provided {@link TraceIdExtractor}
+ * for headers that might contain a trace ID, and provides the first one that is found.
```

Comment:
Still: there is nothing about `headers` in this class.
Just imaging you are me and you don't know what is Stackdriver. So, I'm very confused why are we talking about headers here when `TraceIdExtractor` contract doesn't imply that and doesn't make me to stay only with the headers. 

I could fix during merge, but we don't have permissions to push to the `master` directly.
This way I would like to see the fix or good argument why do we stay just only with headers.

Thanks

Reply:
Ah I see. My apologies I get what you mean,  I will remove the mention of headers.